server/views/applications/add.njk

```diff
@@ -43,7 +43,7 @@
       {% endif %}
       <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
         <div class="govuk-radios__item">
-          <input class="govuk-radios__input" id="testEnvironment" name="environment" type="radio" value="test" aria-controls="conditional-environment"{{ ' checked="checked"' if this_data.environment === 'test' }} />
+          <input class="govuk-radios__input" id="testEnvironment" name="environment" type="radio" value="test"{{ ' checked="checked"' if this_data.environment === 'test' }} />
```

Comment:
I'm guessing you've checked that functionality is still the same without `aria-controls` namely checking the box and the box remains checked once you hit save with errors on the page?

Reply:
Indeed